python/cudf/cudf/core/dataframe.py

```diff
@@ -3271,6 +3271,78 @@ def nsmallest(self, n, columns, keep="first"):
         """
         return self._n_largest_or_smallest(False, n, columns, keep)
 
+    @_cudf_nvtx_annotate
+    def swaplevel(self, i=-2, j=-1, Axis=0):
```

Comment:
```suggestion
    def swaplevel(self, i=-2, j=-1, axis=0):
```

This looks to be the pattern for other methods

Reply:
Sure, I shall update Axis to axis wherever needed.